autoscheduler/frontend/src/components/Empty/Empty.tsx

```diff
@@ -1,11 +1,10 @@
 import * as React from 'react';
 import { RouteComponentProps } from '@reach/router';
 import * as styles from './Empty.css';
-import CourseSelectCard from '../CourseSelectCard/CourseSelectCard';
 
 const Empty: React.FC<RouteComponentProps> = () => (
   <div className={styles.empty}>
-    <CourseSelectCard id={0} />
+    Stuff
```

Comment:
lol forgot that this existed

Reply:
Yeah I didn't notice until changing the props for CourseSelectCard that we still had it. We might as well delete it